lib/services/demoCampus.js

```diff
@@ -0,0 +1,73 @@
+const Schmervice = require('schmervice');
+
+module.exports = class DemoCampusService extends Schmervice.Service {
+    async insertDemoCampus(campus, phone_no) {
+        const { DemoCampus } = this.server.models()
+        try {
+            const demo = await DemoCampus
+                .query()
+                .insert({ campus, phone_no })
+            // return 'data inserted'
+            return demo;
+        } catch (error) {
+            console.log(error);
+            return 'Error is coming...'
+
+        }
+    }
+
+    async findall(limit) {
+        const { DemoCampus } = this.server.models();
+        try {
+            const demo = await DemoCampus
+                .query().limit(limit)
```

Comment:
you have used limit here. In limit if you give 1 it will give the first object in the output. what if I want to see the second one or the third one?